packages/dotcom-build-sass/src/monitored-sass-loader.ts

```diff
@@ -4,8 +4,19 @@ import os from 'os'
 import sassLoader from 'sass-loader'
 
 class SassStats {
-  #noticeThrottle = 10 * 1000
-  #stats = { totalTime: 0, notice: null }
+  #noticeStrategies = ['throttle', 'never', 'always']
+  #noticeStrategy = this.#noticeStrategies.includes(process.env.FT_SASS_STATS_NOTICE)
```

Comment:
**question:** How will these environment variables be set?

Reply:
I intend to work with product teams to set via Doppler for their projects. Run this idea past Umberto who is happy to do that for Storytelling :)

Reply:
Lovely! Makes sense to me.